core/workspace_svg.js

```diff
@@ -906,6 +906,62 @@ Blockly.WorkspaceSvg.prototype.paste = function(xmlBlock) {
 };
 
 /**
+ * Refresh the toolbox unless there's a drag in progress.
+ * @private
+ */
+Blockly.WorkspaceSvg.prototype.refreshToolboxSelection_ = function() {
+  if (this.toolbox_ && this.toolbox_.flyout_ && !this.currentGesture_) {
+    this.toolbox_.refreshSelection();
+  }
+};
+
+/**
+ * Rename a variable by updating its name in the variable list.
+ * TODO: google/blockly:#468
```

Comment:
Yes, I have a follow up PR right after this.
This is not a TODO anymore. It is sprinkled throughout a lot of files so I'll remove all of those in a separate PR and close the issue.